sourcegraph/sourcegraph
# Wildcard: Add progress bar to the wildcard library

Came from this [discussion](https://github.com/sourcegraph/sourcegraph/pull/54416#discussion_r1246518976) about progress

## Todo
- [x] Implement basic layout and animations 
- [ ] Adjust progress colours (@toolmantim @danielmarquespt Since we don't have a good design for this component in particular, I hope you can help me with picking the right colour palette)

## Test plan
- Make sure that colours are right for both dark and light theme
- Make sure that progress layout and animations works properly in all three major browsers (Safari, Chrome, Firefox)
